libs/dh/shared/util-application-insights/src/lib/dh-application-insights.service.ts

```diff
@@ -0,0 +1,95 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Inject, Injectable } from '@angular/core';
+import { Router } from '@angular/router';
+import { AngularPlugin } from '@microsoft/applicationinsights-angularplugin-js';
+import {
+  ApplicationInsights,
+  IExceptionTelemetry,
+  IPageViewTelemetry,
+} from '@microsoft/applicationinsights-web';
+
+import {
+  DhAppEnvironmentConfig,
+  dhAppEnvironmentToken,
+} from '@energinet-datahub/dh/shared/environments';
+
+@Injectable({ providedIn: 'root' })
+export class DhApplicationInsights {
+  private angularPlugin = new AngularPlugin();
+  private appInsights = new ApplicationInsights({
+    config: {
+      instrumentationKey:
+        this.dhAppConfig.applicationInsights.instrumentationKey,
```

Comment:
Question: should we change this to use a `connectionString` instead or create another issue for that?

Reply:
If we can get the `connectionString` the same way we got the `instrumentationKey` in the pipeline, then let's change it. We have time for it in the current task.